package.json

```diff
@@ -34,13 +34,15 @@
       "ie 11"
     ]
   },
-  "dependencies": {
+  "devDependencies": {
     "@navikt/bedriftsmeny": "^2.2.0",
-    "@navikt/fnrvalidator": "^1.1.2",
-    "@navikt/helse-arbeidsgiver-felles-frontend": "^0.1.29",
+    "@navikt/fnrvalidator": "^1.1.3",
```

Comment:
Disse er vel dependencies? Jeg var veldig sikker på det frem til lunsj i dag, men nå vet jeg ikke lenger?

Reply:
Betyr ikke dependencies det som trengs for å kjøre i prod? Vi trenger ingenting i prod når det kjøres fra html side?